src/Payments.sol

```diff
@@ -47,6 +47,20 @@ contract Payments is
 
     uint256 public constant PAYMENT_FEE_BPS = 10; //(0.1 % fee)
 
+    // Events
+    event AccountLockupSettled(address indexed token, address indexed owner, uint256 lockupCurrent, uint256 lockupRate, uint256 lockupLastSettledAt);
+    event OperatorApprovalSet(address indexed token, address indexed client, address indexed operator, bool approved);
+
+    event RailCreated(uint256 indexed railId);
```

Comment:
Upto you but would be nice to include payer and payee on this. But I guess you can always get that with a getter.